File: Python/stage2/S2_ARR_04.py
# S2_ARR_04 — Flatten one level
def flatten1(arr):
   result = []
   for item in arr:
       if isinstance(item, list):
           result.extend(item)
       else:
           result.append(item)
   return result
print(flatten1([1, [2,3], [4], 5]))

File: js/stage2/S2_ARR_04.js
// S2_ARR_04 — Flatten one level
function flatten1(arr) {
  const result = [];
  for (const item of arr) {
    if (Array.isArray(item)) {
      result.push(...item);
    } else {
      result.push(item);
    }
  }
  return result;
}
// Test
console.log(flatten1([1, [2, 3], [4], 5]));
